### PR TITLE
Add contact section

### PR DESCRIPTION
**Pull request by lewisjn-arch**
This PR has the following:

- [ ] Added contact me section.
- [ ] Added the user input and text area section.
- [ ] Improved button functionality with the hover feature.
- [ ] Styled the header with the position fixed property for better user experience.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -223,8 +223,30 @@ <h3 class="category">Skills</h3>
...
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </h3>
+            <input type="text" name="name" placeholder="Full Name" id="name-id" minlength="4" maxlength="30" required>
+            <div class="seperation_line"></div>
+            <input type="email" name="email" placeholder="Email address" id="email-id" required>
+            <div class="seperation_line"></div>
+            <textarea name="text-area" placeholder="Write me something..." id="text-area" rows="5" cols="30" maxlength="500" required></textarea>
+            <input type="submit" name="Get in touch" id="submit" value="Get in touch" placeholder="Get in touch">
+            <div class="seperate_line_2"></div>
+        </form>
```
Comment (ghost) [inline index.html]: - [ ] Good work up to this point but there are some aspects you can improve on. Your input elements appear to be very close compared to the original design. Kindly increase the margin between them

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Form submission works correctly👍 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (lewisjn-arch) [reply index.html]: Thank you. I will get to it right away.

**Comment by lewisjn-arch** [comment]
This second review request has the following:
 
- [ ] I have added a margin between the two user input fields to match the design.
- [ ] I have edited the placeholder text colors to match the design template.

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @lewisjn-arch 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**